pkg/controller/v1beta1/inferenceservice/utils/utils.go

```diff
@@ -85,18 +84,31 @@ func IsMemoryResourceAvailable(isvc *v1beta1api.InferenceService, totalReqMemory
 	return predictorMemoryLimit.Cmp(totalReqMemory) >= 0
 }
 
+func getModelNameFromArgs(args []string) string {
+	modelName := ""
+	for i, arg := range args {
+		if strings.HasPrefix(arg, constants.ArgumentModelName) {
+			// Case 1: ["--model-name=<model-name>"]
+			modelNameValueArr := strings.Split(arg, "=")
+			if len(modelNameValueArr) == 2 {
+				modelName = modelNameValueArr[1]
```

Comment:
```suggestion
				modelName = modelNameValueArr[1]
				return modelName
```

Reply:
When both formats exist, do we expect to get the model name from the first or last arg?

Reply:
from the test looks like you are expecting to get the model name from the last arg

Reply:
yeah, that's how python's argparse behaves.